### DISCUSSION TITLE
Windows September Update KB5030219 causes crashing.

### DISCUSSION THREAD

**Discussion by slapshotkc** [valinet/ExplorerPatcher#2076] [General]
Ran the monthly updates last night and shut down.  This morning I had to resort to removing EP from my system.  Using latest version, 22621.1992.56.3.

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/2051

Please close this one.

Possible fixes:

1. In ExplorerPatcher Properties, go to Advanced and tick **Do not change the taskbar context menu**
2. In ExplorerPatcher Properties, go to File Explorer and untick **Disable the windows 11 context menu**
3. In ExplorerPatcher Properties, go to Taskbar and set **Taskbar style** to **Windows 11**

**Reply by slapshotkc** [reply]
Did a search for KB5030219, did not find. Will just abandon it then, Taskbar style to Windows 11 makes this useless for me, I HATE the Windows 11 taskbar.

**Reply by aldeniangman** [reply]
why won't Vailnet fix our problems and DUMB Microsoft preventing the use of using of explorerpatcher on the insider previews because they say its not safe (Microsoft plz add these features)
